garrysmod/gamemodes/terrortown/gamemode/vgui/sb_row.lua

```diff
@@ -320,6 +320,13 @@ function PANEL:DoRightClick()
    menu:Open()
 end
 
+local frame

```

Comment:
This approach with the global variable does not handle situation where multiple voice volume panels are open.